bundles/org.openhab.core.thing.xml/src/main/java/org/eclipse/smarthome/core/thing/xml/internal/ThingTypeXmlProvider.java

```diff
@@ -131,34 +131,46 @@ private void addConfigDescription(ConfigDescription configDescription) {
     public synchronized void addingFinished() {
         Map<String, ChannelType> channelTypes = new HashMap<>(10);
```

Comment:
What is the purpose of this `Map`?

Reply:
It doesn't seem to have any use so I'll remove it. :+1: